core/vehicle/adapter.go

```diff
@@ -0,0 +1,83 @@
+package vehicle
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/evcc-io/evcc/api"
+	"github.com/evcc-io/evcc/core/keys"
+	"github.com/evcc-io/evcc/server/db/settings"
+	"github.com/evcc-io/evcc/util"
+)
+
+var _ API = (*adapter)(nil)
+
+type adapter struct {
+	log         *util.Logger
+	name        string
+	api.Vehicle // TODO handle instance updates
+}
+
+func (v *adapter) key() string {
+	return fmt.Sprintf("vehicle.%s.", v.name)
+}
+
+func (v *adapter) Name() string {
+	return v.name
+}
+
+// GetMinSoc returns the min soc
+func (v *adapter) GetMinSoc() int {
+	if v, err := settings.Int(v.key() + keys.MinSoc); err == nil {
+		return int(v)
+	}
+	return 0
+}
+
+// SetMinSoc sets the min soc
+func (v *adapter) SetMinSoc(soc int) {
+	v.log.DEBUG.Printf("set %s min soc: %d", v.name, soc)
+	settings.SetInt(v.key()+keys.MinSoc, int64(soc))
```

Comment:
@andig Hier müssen die vehicles neu gepublisht werden damit der Wert in der UI sichtbar wird. Analog bei Limit und Plan

Reply:
Bäh, das ist doof. Dafür müsste der Loadpoint an die Site ran kommen. Als workaround jetzt auch einmal pro Durchlauf, da muss noch was besseres her.